tools/bootstrap-librdkafka.sh

```diff
@@ -29,7 +29,7 @@ mkdir -p "$BUILDDIR/librdkafka"
 pushd "$BUILDDIR/librdkafka"
 
 test -f configure ||
-curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" | \
+curl -q -L "https://github.com/confluentinc/librdkafka/archive/refs/tags/v${VERSION}.tar.gz" | \
```

Comment:
This "v" breaks Travis build, use the tag containing "v" as version

Reply:
Thanks, I missed checking this. Have updated it now.